versioned_docs/version-0.x.0/kubeslice-cli/reference/configuration-parameters.mdx

```diff
@@ -0,0 +1,85 @@
+# Configuration Parameters
+This topic describes the parameters used in creating the topology YAML file to install opensource KubeSlice.
+
+## Cluster Configuration Parameters
+The following table describes the cluster configuration parameters used to create the topology YAML.
+
+|Parameter|Parameter Type|Description|Required|
+|----|----|----|----|
+|profile|String|It is the KubeSlice profile to create demo clusters.The support values are **minimal-demo** and **full-demo**|Mandatory|
+|kube_config_path|String|It is the absolute path to the kubeconfig file to use topology setup; topology only| Optional|
```

Comment:
What is  topology only? Is it the value? If yes, then describe it as the value of the parameter...

Reply:
updated